models/nxos/lldp/replaced_example_01.txt

```diff
@@ -0,0 +1,23 @@
+# Using replaced
+# Before state:
+# -------------
+#
+#feature lldp
+#lldp timer 35
+#lldp holdtime 100
+#lldp reinit 5
+
+- name: Replace device configuration of specific LLDP attributes with provided configuration
+  nxos_lldp:
+    config: 
+      enable: true
+      timer: 40
+    state: replaced
+
+# After state:
```

Comment:
Replace seems to not work as expected. Based on the task, the only lldp config lines present should be `feature lldp` and `lldp timer 40`. Other configurations should be negated.

Reply:
Yes.